deploy-manage/users-roles/cluster-or-deployment-auth/user-roles.md

```diff
@@ -77,12 +60,41 @@ The authorization process revolves around the following constructs:
 
 A role has a unique name and identifies a set of permissions that translate to privileges on resources. You can associate a user or group with an arbitrary number of roles. When you map roles to groups, the roles of a user in that group are the combination of the roles assigned to that group and the roles assigned to that user. Likewise, the total set of permissions that a user has is defined by the union of the permissions in all its roles.
 
-The method for assigning roles to users varies depending on which realms you use to authenticate users. For more information, see [Mapping users and groups to roles](mapping-users-groups-to-roles.md).
+## Set up user authorization using RBAC
 
+Review these topics to learn how to configure RBAC in your cluster or deployment:
+
+* Learn about [built-in roles](/deploy-manage/users-roles/cluster-or-deployment-auth/built-in-roles.md)
+* [Define your own roles](/deploy-manage/users-roles/cluster-or-deployment-auth/defining-roles.md)
+* Learn about the [Elasticsearch](asciidocalypse://docs/elasticsearch/docs/reference/elasticsearch/security-privileges.md) and [Kibana](/deploy-manage/users-roles/cluster-or-deployment-auth/kibana-privileges.md) privileges you can assign to roles
```

Comment:
The Elasticsearch privileges link doesn't work.
It points to asciidocalypse but I see `elasticsearch-privileges.md` available in this content.

Reply:
I had taken it out because it's duplicated in the reference, but put it back. good catch.